girder/__init__.py

```diff
@@ -136,13 +138,12 @@ def getLogPaths():
     }
 
 
-def _setupLogger():
+def _attachFileLogHandlers():
```

Comment:
Is this function expected to be called multiple times? There is some logic within that removes all handlers from the main Girder logger, but I wonder if it's necessary:
https://github.com/girder/girder/blob/0c28e5be9f5eaee1f5220930d142908818d2b234/girder/__init__.py#L180-L185

If tests need to reset / remove loggers, they should do so explicitly.

Reply:
I'll remove that logic and see what breaks :)